src/components/ToastCard/ToastCard.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { mount } from "enzyme";
+
+import ToastCard from "./ToastCard";
+
+describe("Toast Card", () => {
+  const t = {
```

Comment:
This object is shared between all of the test instances. While unlikely in this case it's a good idea to always clone the data for each test so that you don't end up with cascading or spurious failures. In this case it's an easy fix, just use the `cloneDeep` library we have and clone it for each test.